src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -216,7 +216,9 @@
         "httpMetrics": true,
         "systemMetrics": true
     },
-    "AzureMonitor": {
+    "Telemetry": {
+        "Provider": "ApplicationInsights",
+        "InstrumentationKey": "",
```

Comment:
We'll need to update the template too:

https://github.com/microsoft/fhir-server/blob/cc5576f4a90ead59f5d5b15341d5a76b754da506/samples/templates/default-azuredeploy-docker.json#L361

Will existing deployments also break?

Reply:
The PR pipeline with the template change passed. I am resolving the conversation. Please let me know if more testing needs to be done. Thank you.